### ISSUE TITLE
[Keadex Mina Bug] Newlines are rendered as plaintext

### ISSUE THREAD

**Issue by piorus** [keadex/web-apps#8]
**Describe the bug**
Hey, I've found out about Mina today, and it looks pretty cool but there is a discrepancy between it, and PlantUML way rendering of newlines. Mina renders them in a plaintext, and PlantUML renders them as correctly - as newlines.

**To Reproduce**

use it as a diagram:

```
@startuml

Person(personAlias, "Label", "Optional\nDescription")

Container(containerAlias, "Label", "Technology", "Optional\nDescription")

System(systemAlias, "Label", "Optional\nDescription")

Rel(personAlias, containerAlias, "Label", "Optional\nTechnology")

@enduml
```

**Expected behavior**
Newlines are rendered properly, as new lines, not as `\n`

**Screenshots**

PlantUML VSCode plugin with code:
```
@startuml C4_Elements
!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml

Person(personAlias, "Label", "Optional\nDescription")
Container(containerAlias, "Label", "Technology", "Optional\nDescription")
System(systemAlias, "Label", "Optional\nDescription")

Rel(personAlias, containerAlias, "Label", "Optional\nTechnology")
@enduml
```

outputs:
![](https://i.imgur.com/5xKhjiA.png)

and Keadex Mina renders:
![](https://i.imgur.com/Y9kENnU.png)

**App Details:**
{
  "version": "1.14.0",
  "tauriVersion": "2.0.6",
  "arch": "x86_64",
  "platform": "linux",
  "osType": "linux",
  "kernelVersion": "22.4.0"
}

**Comment by keadex** [comment]
@piorus issue solved in the latest release: [Mina 1.14.1](https://github.com/keadex/keadex/releases/tag/keadex-mina-v1.14.1)

**Comment by piorus** [comment]
marvelous, thank you!
